shared/util/feature-flags.desktop.tsx

```diff
@@ -15,6 +15,7 @@ const ff: FeatureFlags = {
   chatIndexProfilingEnabled: false,
   conflictResolution: false,
   dbCleanEnabled: false,
+  fastAccountSwitch: true,
```

Comment:
why have this flag and just turn it on?

Reply:
Since release is soon and there are a lot of edge cases that felt easy to mess up.

Reply:
no i mean, why is this true. this means its on for everyone right now

Reply:
I want people to test it - I think it is in a working state and I think I have fixed all the edge-case bugs (though Y2K-154 still should get looked at even if I haven't been able to trigger it).

Reply:
That's what admin is for

Reply:
admin doesn't work on mobile right?

Reply:
This isn't a mobile file

Reply:
ok, I'll turn off the flag on desktop and put it in admin override.